lib/core/src/display/shape/text/glyph/msdf.rs

```diff
@@ -126,15 +124,14 @@ mod test {
     }
 
     #[wasm_bindgen_test(async)]
-    fn msdf_translation_converting() -> impl Future<Output=()> {
-        TestAfterInit::schedule(|| {
-            let mut msdf = MultichannelSignedDistanceField::mock_results();
-            msdf.translation = Vector2::new(16.0, 4.0);
+    async fn msdf_translation_converting() {
+        basegl_core_msdf_sys::initialized().await;
+        let mut msdf = MultichannelSignedDistanceField::mock_results();
+        msdf.translation = Vector2::new(16.0, 4.0);
 
-            let converted = convert_msdf_translation(&msdf);
-            let expected = nalgebra::Vector2::new(0.5, 1.0/8.0);
+        let converted = convert_msdf_translation(&msdf);
+        let expected = nalgebra::Vector2::new(0.5, 1.0/8.0);
```

Comment:
I know this is old code, but these `8` and `16` should be constants. I dont understand where they come from.

Reply:
And the variable name "expected" does not explain, that these are just expected values?